Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Rapid7_Enrichment.yml

```diff
@@ -853,12 +853,12 @@ tasks:
     isautoswitchedtoquietmode: false
   "27":
     id: "27"
-    taskid: ab2e86ed-b0b3-49a3-85d0-a4ee5db2762d
+    taskid: 1ab733c9-9298-4e95-8ad5-0aaa4b8aba6b
     type: regular
     task:
-      id: ab2e86ed-b0b3-49a3-85d0-a4ee5db2762d
+      id: 1ab733c9-9298-4e95-8ad5-0aaa4b8aba6b
       version: -1
-      name: Set system IDs grid field (Asset ID)
+      name: Set system IDs grid field (asset ID)
```

Comment:
General comment: anywhere were task names are changes, we should update the screenshot of the playbook (light mode), please

Reply:
resolved